src/sentry/search/events/fields.py

```diff
@@ -2758,6 +2759,15 @@ def resolve_orderby(self, orderby: Optional[Union[List[str], str]]) -> List[Orde
                     validated.append(OrderBy(selected_column, direction))
                     continue
 
+                elif (
```

Comment:
oh nice! do we also want to update `_resolve_transaction_status` and `_resolve_error_handled_alias` as a part of this?

Reply:
Good call, updated!